logger/logger.go

```diff
@@ -0,0 +1,71 @@
+package logger
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/databricks/databricks-sdk-go/logger"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+)
+
+type TfLogger struct {
+	Name  string
+	Level logger.Level
+}
+
+func (tfLogger *TfLogger) Enabled(_ context.Context, level logger.Level) bool {
+	return true
+}
+
+func (tfLogger *TfLogger) Tracef(ctx context.Context, format string, v ...any) {
+	if tfLogger == nil {
+		tflog.Trace(ctx, format, convertToMap(v))
```

Comment:
I don't think this is the right way to do this. IIUC, the extra parameters are meant to be interpolated into the provided format string. This will leave the format string unprocessed and log the parameters as extra k-v pairs. Instead, you should use fmt.Sprintf to format the string.